portals/publisher/src/main/webapp/site/public/conf/settings.json

```diff
@@ -42,7 +42,7 @@
         },
         "loadDefaultLocales": false,
         "supportedDocTypes": "application/vnd.openxmlformats-officedocument.wordprocessingml.document, application/msword, application/pdf, text/plain, application/vnd.ms-excel, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, application/vnd.oasis.opendocument.text, application/vnd.oasis.opendocument.spreadsheet, application/json, application/x-yaml, .yaml, .md",
-        "docUrl": "https://apim.docs.wso2.com/en/4.6.0/",
+        "docUrl": "https://apim.docs.wso2.com/en/latest/",
```

Comment:
@ranuka-laksika This needs to be kept version-specific (4.6.0), as each release is required to point to its respective documentation space.